gateway-ha/src/test/java/io/trino/gateway/baseapp/TestAppModule.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.gateway.baseapp;
+
+import io.dropwizard.core.server.DefaultServerFactory;
+import io.dropwizard.core.setup.Environment;
+import io.dropwizard.jetty.HttpConnectorFactory;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import java.util.Collections;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.Mockito.when;
+
+public class TestAppModule
```

Comment:
This test passes even before changing AppModule. It means the change isn't covered.

Reply:
Any chance you can verify @ebyhr .. @prakhar10 pushed an update to the test .. I wont have time to verify

Reply:
I confirmed the test still doesn't cover the change.

Reply:
Sorry earlier changes didnt get pushed for some reason, please check now.